### DISCUSSION TITLE
How to save it in a folder in the github branch

### DISCUSSION THREAD

**Discussion by RouJiANG-14** [gildas-lormeau/SingleFile#1195] [Q&A]
I tried the method of "branch name/file name", but it was not successful,
For example: "main/singlefile", "main/singlefile/", "/singlefile"
They are not working
Excuse me, is there any way?
Hope to get your answer.

**Comment by gildas-lormeau** [comment]
Add the folder name in the `File > name template` option, e.g. `singlefile/{page-title} ({date-locale} {time-locale}).html`.

**Comment by RouJiANG-14** [comment]
!!!!
Thank you very much! You resolved my doubts!！！

**Comment by gildas-lormeau** [comment]
You're welcome :)
